src/main/resources/assets/robotics/lang/de_de.json

```
@@ -27,22 +27,22 @@
   "item.robotics.extend_o_boots.tooltip": "EXTEND-O-BOOTS",
   "item.robotics.extend_o_boots.tooltip.summary": "Dein _Anziehbares Gerüst_! Erstellt ein _einstellbares temporäres Gerüst_ unter dir.",
   "item.robotics.extend_o_boots.tooltip.control1": "Halte STRG wärend du es trägst",
-  "item.robotics.extend_o_boots.tooltip.action1": "_Aktiviere_ sie durch _scrollen_",
+  "item.robotics.extend_o_boots.tooltip.action1": "_Steuere_ es durch _scrollen_",
 
   "item.robotics.program.tooltip": "PROGRAMM",
-  "item.robotics.program.tooltip.summary": "Ein _Programm_ das code enthält! Lässt dich _Roboter programieren_.",
-  "item.robotics.program.tooltip.control1": "R-KLicke den Programm Schreiber",
+  "item.robotics.program.tooltip.summary": "Ein _Programm_ das Code enthält! Lässt dich _Roboter programieren_.",
+  "item.robotics.program.tooltip.control1": "R-KLicke den Programm-Schreiber",
   "item.robotics.program.tooltip.action1": "Ermöglicht es dir, den _Inhalt_ des Programms zu _bearbeiten_",
   "item.robotics.program.tooltip.control2": "R-Klicke einen programmierbaren Roboter",
-  "item.robotics.program.tooltip.action2": "Ermöglicht dir, den _Code_ des Programms auf den _Roboter_ anzuwenden",
+  "item.robotics.program.tooltip.action2": "Ermöglicht es dir, den _Code_ des Programms auf den _Roboter_ anzuwenden",
 
-  "block.robotics.code_editor.tooltip": "PROGRAMM SCHREIBER",
+  "block.robotics.code_editor.tooltip": "PROGRAMM-SCHREIBER",
   "block.robotics.code_editor.tooltip.summary": "Ein Block, mit dem du den _Inhalt_ eines Programms _bearbeiten_ kannst!",
   "block.robotics.code_editor.tooltip.condition1": "Bei R-Klick mit einem Programm",
   "block.robotics.code_editor.tooltip.behaviour1": "Lässt dich das _Programm verändern_",
 
-  "robotics.ponder.programming.header": "Programiern mit einem Programm Schreiber.",
-  "robotics.ponder.programming.text_1": "Um den Inhalt eines Programmes zu ändern brauchst du einen Programm Schreiber.",
-  "robotics.ponder.programming.text_2": "R-Klicke den Programm Schreiber mit dem Programm um zu starten!",
-  "robotics.ponder.programming.text_3": "Wenn du mit dem programieren zu ende bist, rechts-klicke einen programmierbaren Roboter mit dem Programm."
-}
\ No newline at end of file
+  "robotics.ponder.programming.header": "Programiern mit einem Programm-Schreiber.",
+  "robotics.ponder.programming.text_1": "Um den Inhalt eines Programmes zu ändern brauchst du einen Programm-Schreiber.",
+  "robotics.ponder.programming.text_2": "R-Klicke den Programm-Schreiber mit dem Programm, um zu starten!",
+  "robotics.ponder.programming.text_3": "Wenn du mit dem Programieren fertig bist, R-klicke einen programmierbaren Roboter mit dem Programm, um es zu testen."

```

The word `testen` doesn't fit there as you use the program to start it, not to test it.
I'd suggest switching `testen` with `starten`:
```suggestion
  "robotics.ponder.programming.text_3": "Wenn du mit dem Programieren fertig bist, R-klicke einen programmierbaren Roboter mit dem Programm, um es zu starten."
```